Source/Other/Extensions/CollectionExtension.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  CollectionExtension.swift
```

Comment:
Is this file related to this linting pod PR? Seems unrelated?

Reply:
Ahhh I see your comment
Still would split to a separate PR. Why introduce an extension nothing's using?

Reply:
Swiftlint would still complain... :) it's realted.